packages/oc-pages/dashboard/components/cells/deployment-controls.vue

```diff
@@ -84,8 +84,13 @@ export default {
                 result.push('view-artifacts')
             } 
 
-            if(!this.deploymentItem?.isUndeployed) result.push('local-deploy')
-            result.push('view-in-repository')
+            if(window.gon.projectId) {
+                //temporary limitation (restrict to dashboard app)
```

Comment:
The menu option to locally deploy inside of "Your Deployments" was listed, but didn't work
This check just keeps it out of the list

What I'll do next is provide the `projectId` as a prop through this component down to the dropdown menu component.  If `deployment-controls.vue` has access to `projectId` it can determine whether we have enough information to display the local deploy dialog.

Reply:
oh the "Your Deployments" table on the blue overview page? good catch, i don't think this case is that important -- though i suppose the menu item could redirect to the deployment page and then show the modal there